WalletWasabi/BitcoinCore/Monitoring/RpcStatus.cs

```diff
@@ -16,21 +16,21 @@ private RpcStatus(bool success, ulong headers, ulong blocks, int peersCount)
 				var diff = headers - blocks;
 				if (peersCount == 0)
 				{
-					Status = "Bitcoin Core is connecting...";
+					Status = "Bitcoin Knots is connecting...";
```

Comment:
I wonder if it will be useful to write some code to identify the client type based on agent subversion

Reply:
Let's call it full node as we don't know if it's Knots or Core that's being used by the user.